include/dlaf/eigensolver/band_to_tridiag/mc.h

```diff
@@ -89,15 +94,92 @@ void applyHHRight(const SizeType m, const SizeType n, const T tau, const T* v, T
   blas::ger(ColMaj, m, n, -tau, w, 1, v, 1, a, lda);
 }
 
+// split versions of the previous operations
```

Comment:
I'm ignorant as usual about the algorithms, but "split" says nothing to me in this context. Is that something one knows if one knows how the algorithm works? If yes, this is probably fine. If it refers to something specifically in the way you've implemented the algorithm would you mind briefly explaining what split refers to?

Reply:
This is the kind of comment I'm looking for. 👍🏻

Reply:
Is it better now?

Reply:
Yes, much better. Thank you!